### PR TITLE
File state line ending fixes

### PR DESCRIPTION
**Pull request by dwoz**
### What does this PR do?

Fix tests in `integration.states.test_file`

### Tests written?

No - Fixing tests

### Commits signed with GPG?

Yes

### CODE REVIEW

File: salt/modules/file.py [hunk 1]
```diff
@@ -4080,7 +4080,12 @@ def _get_local_file_source_sum(path):
                 urlparsed_source.netloc, urlparsed_source.path).rstrip(os.sep)
         unix_local_source = parsed_scheme in ('file', '')
```
Comment (gtmanfred) [inline salt/modules/file.py]: is this line still needed?

**Review by gtmanfred — APPROVED** [review]
I would also like for @terminalmage to just double check this.

File: salt/modules/file.py (hunk 1 above)
Comment (dwoz) [reply salt/modules/file.py]: Yes, It get's used later in the method.

Comment (gtmanfred) [reply salt/modules/file.py]: cool

**Review by terminalmage — APPROVED** [review]

**Pull request merged**